docs/conf/prague/2020/welcome-wagon.rst

```diff
@@ -1,15 +1,256 @@
+with both the information on this page, and events we run during the conference.
 
-Schedule
+On this page you'll find important stuff that will help you navigate the conference like a pro,
+make you feel more at home, and help you to manage the constant flow of information.
+
+.. image:: /_static/conf/images/pics/welcome-wagon.jpg
+
+Dururing the conference, we'll connect you with people - both veterans and other first-timers.
+And we'll give you strategies and tips so you can make the most of the conference.
+
+The `FAQs <#faq>`_ should have answers to most of your questions, but let us know if there is anything you'd like to know that isn't there. You can ask us questions in the :doc:`#wtd-conferences channel on Slack </slack>`.
+
+Who we are
+-----------
+
+Hi - we're Beth and Daniel, your Welcome Wagon coordinators! We're really glad you're coming to the {{year}} Write the Docs conference.
+
+If you need a hand at all, we're here to help.
+
+Welcome Wagon events
+--------------------
+
+**Write the Docs Introduction**
+
+*{{date.day_two.dotw}}, {{date.day_two.date}} at 17:00 in the downstairs foyer (near the cloakroom) at Auto-Klub*
+
+Join us for an informal introduction to Write the Docs, to the Welcome Wagon, and to other first-time conference attendees. We'll pass on some information about the conference specifically for first-timers and give everyone a chance to meet someone new before we join the opening reception.
+
+**Venue Tours**
+
+* *{{date.day_two.dotw}}, {{date.day_two.date}} at 17:30 starting near the registration desk at Auto-Klub*
+* *{{date.day_three.dotw}}, {{date.day_three.date}} at 9:15 starting near the registration desk at Auto-Klub*
+
+Come on a short tour of the venue with a veteran Write the Docs attendee so you'll know where everything is and everything you can take part in.
+
+Pro tips
 --------
 
+-  You don't need to go to every talk. Look through the schedule of
+   events before you arrive or while you are eating or taking a break.
+   Figure out which talks you want to see the most. Spread out your time
+   between talks, unconference sessions, networking, and breaks.
+-  Speaking of breaks--conferences are exhilarating, but can also be
+   exhausting. Give your brain a break! Grab a spot in the quiet room
+   or take a quick walk. Play a board game on your lunch break. Come
+   back invigorated.
+-  Starting Monday morning, check the unconference schedule in the unconference
+   room to see if there are any sessions you are interested in
+   attending. New sessions are added all the time, so check back
+   periodically.
+-  Eat! You can use the energy.
+-  Are you looking for a job or is there an opening at your company?
+   Check out the job board in the unconference room.
+
+
+FAQ
+---
+
+.. contents::
+   :local:
+
+Where is everything?
+~~~~~~~~~~~~~~~~~~~~
+
+The conference main stage is in the Auto Klub on the second floor. The unconference
```

Comment:
No it's not ;)
Quite a few other Auto Klub references in the rest of this document too.